pyccel/codegen/printing/ccode.py

```diff
@@ -63,7 +63,9 @@
                   ('complex',4) : 'float complex',
                   ('int',4)     : 'int',
                   ('int',8)     : 'long',
-                  ('bool',1)    : 'int'}
+                  ('int',2)     : 'int',
+                  ('int',1)     : 'int',
+                  ('bool',4)    : 'int'}
```

Comment:
Would you mind fixing issue #388 too? As we are using a dictionary we should either check whether the key we are looking for is in the dictionary or use a try/except block to catch the `KeyError`.
This will be needed here:
https://github.com/pyccel/pyccel/blob/f20898be8d057448d2c41dd5eb1d3ec7027d2cae/pyccel/codegen/printing/ccode.py#L201
and possibly in other places too.

When the error has been caught/when the key is not in the dictionary we need to call the following code:
```
errors.report(PYCCEL_RESTRICTION_TODO, symbol=expr,
                severity='fatal')
```

Reply:
I didn,t use the suggested print for boolean value 
```
   'bool' : 'PyBool_FromLong({})',
   'int' : '{}',
   'double' : '{}'
}
code = ", ".join(pytype_printer_registry['str_dtype(x)].format(x)  for x  in expr list' )
```
because  im working in complex datatypes too so i wanted to look for a  general code form to print both complpex and boolean type.